proto/nexus/v1beta1/query.proto

```diff
@@ -22,3 +23,15 @@ message LatestDepositAddressRequest {
 }
 
 message LatestDepositAddressResponse { string deposit_addr = 1; };
+
+// TransfersForChainRequest represents a message that queries the
+// transfers for the specified chain
+message TransfersForChainRequest {
+  string chain = 1;
+  exported.v1beta1.TransferState state = 2;
+}
+
+message TransfersForChainResponse {
+  repeated exported.v1beta1.CrossChainTransfer transfers = 1
```

Comment:
I also originally used a custom protobuf to return the transfers because most of the fields of the `exported.v1beta1.CrossChainTransfer` type would look weird and some would be redundant, like the Transfer Type field. Unless the microservice does need the full `exported.v1beta1.CrossChainTransfer` type?

Reply:
the response values were literally the same values, just as strings as in `CrossChainTransfer`

Reply:
Not quite, because the response now will contain the entire `nexus.Chain` struct. I am not sure we need that much information.

Reply:
ok fair point, I didn't realize it was such a big structure

Reply:
@jcs47 I don't think for now the chain object is a big issue and we need this in, so if you feel strongly about it, please open another issue/PR to create a leaner response struct 🙂